DylibBinder/StringExtensions.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace DylibBinder {
+	internal static class StringExtensions {
+		static readonly string [] BuiltInTypes = { "Bool", "Double", "Float", "UInt", "Int" };
+
+		internal static bool IsPublic (this string s)
+		{
+			return !s.Contains ("_");
```

Comment:
Using a char is slightly faster:

```suggestion
			return !s.Contains ('_');
```

Reply:
Hmm when I try to use `return !s.Contains ('_');` I get an error saying that it cannot convert from 'char' to 'string'
and it asks if I want to include Linq and use it's Contains which I don't think is what you are saying correct?

i do see in the docs that string.Contains should contain the Contains (char c) [here](https://docs.microsoft.com/en-us/dotnet/api/system.string.contains?view=net-5.0#System_String_Contains_System_Char_) though so I am not sure what I am missing?

Reply:
![image](https://user-images.githubusercontent.com/50846373/118021520-ef4c1c00-b320-11eb-9151-8b540fd35088.png)
CS1503: Argument 1: Cannot convert from 'char' to 'string'

Reply:
@tj-devel709 is right, the char overload is not available in .NET Framework 4.8 https://docs.microsoft.com/en-us/dotnet/api/system.string.contains?view=netframework-4.8

Reply:
Ah, that makes sense.

The old way of doing this was to call `string.IndexOf (char) >= 0`(https://docs.microsoft.com/en-us/dotnet/api/system.string.indexof?view=netframework-4.8#System_String_IndexOf_System_Char_)

No need to change this, the current code is more legible.